source/Mission.cpp

```diff
@@ -209,6 +211,8 @@ void Mission::Load(const DataNode &node)
 		}
 		else if(child.Token(0) == "infiltrating")
 			hasFullClearance = false;
+		else if(child.Token(0) == "service")
```

Comment:
This PR introduces 2 new features; the ability to have certain actions trigger "daily/every day", and the option to mark some missions as a "service provided to the player".

Those features don't need each-other, and I would recommend to split this into 2 separate PRs. The daily actions sounds like something we should be able to merge as soon as we agree on the actual keyword.

I'm not sure yet what we should do with the "service" idea. Adding the concept of "a service offered to the player" is a lot more fundamental than adding a daily trigger, there are quite a few directions we could go there. So I expect a bit longer review-cycle with some more architecture/design-work for this part.

Reply:
So you want me to make an issue for the service tag and leave this PR only for "on daily"

Reply:
> So you want me to make an issue for the service tag and leave this PR only for "on daily"

Yes

Reply:
Ok i have done so:  #7056